src/components/PageSection.tsx

```diff
@@ -11,10 +11,12 @@ function PageSectionComponent(props: Props) {
   return (
     <MainContainer>
       {props.position !== 'right' && <ImageContainer image={props.image} />}
+      {props.position !== 'right' && <BufferContainer />}
       <TextContainer>
         <Title>{props.title}</Title>
         <Body>{props.description}</Body>
       </TextContainer>
+      {props.position === 'right' && <BufferContainer />}
```

Comment:
can consider grouping both conditional lines into one i.e. props.position === 'right' && \<BufferContainer/\>\<ImageContainer/\> (?)

Reply:
I think I did this previously but Chun Yu suggested the current structure cuz it has less code. So shd I change to this?

Reply:
hmm it's up to u which you prefer tho, i'm just chipping in my take on simplifying this. although you could also consider grouping the `props.position === 'right'` so that you only need to write once